src/components/ITSBTooltip/ITSBTooltip.js

```diff
@@ -1,3 +1,66 @@
-export const ITSBTooltip = ({ object }) => {
-  return object?.properties?.title;
-};
+import { sortWaypointsByTime } from '../ITSBStore';
+
+/**
+ * Format the date by moving the year to the end and changing dash delimiters
+ * to slashes; or return a question mark if there is no date.
+ *
+ * @param {string} date The date as a string formatted YYYY-MM-DD.
+ * @returns {string} Date formatted MM/DD/YYYY.
+ */
+function fmt(date) {
+  const split = date?.split('-');
+  if (split?.length) {
+    split.push(split.shift());
+  }
+  return split?.join('/') || '?';
+}
+
```

Comment:
I think you can just re-use `formatInterval(waypoint)` from `components/ITSBStore/utils/waypointUtils.js`.